variables.tf

```diff
@@ -55,12 +55,37 @@ variable "multinode_vm_subnet" {
   type = string
 }
 
-variable "compute_is_vm" {
-  description = "Should be set to true if VMs are being used for compute nodes"
-  type = bool
+variable "compute_disk_size" {
+  description = "Block storage root disk size for compute nodes in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 100
 }
 
-variable "controller_is_vm" {
-  description = "Should be set to true if VMs are being used for controller nodes"
-  type = bool
+variable "controller_disk_size" {
+  description = "Block storage root disk size for controller nodes in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 100
+}
+
+variable "ansible_control_disk_size" {
+  description = "Block storage root disk size for the ansible control node in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 50
+}
+
+variable "seed_disk_size" {
+  description = "Block storage root disk size for the seed node in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 100
+}
+
+variable "storage_disk_size" {
+  description = "Block storage root disk size for storage nodes in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 100
 }
+
+variable "enable_monitoring" {
+  type = bool
+  default = true
```

Comment:
Could benefit from a description.

Reply:
I'm tempted to just remove the variable entirely and simply enable it. I just don't know if there is a downside to having the group declaration there and not using it

Reply:
I've gone ahead and just changed the group declaration permanently. The monitoring functionality can be handled easily enough by just enabling/disabling services in the kayobe configuration